src/backend/api/Fusion.Resources.Api/Fusion.Resources.Api.csproj

```diff
@@ -18,31 +18,31 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="FluentValidation.AspNetCore" Version="8.6.2" />
-    <PackageReference Include="Bogus" Version="30.0.4" />
-
-    <PackageReference Include="Fusion.Integration" Version="3.5.8" />
-    <PackageReference Include="Fusion.Integration.Profile.Abstractions" Version="3.5.4" />
-    <PackageReference Include="Fusion.Integration.Authorization" Version="3.2.1" />
-    <PackageReference Include="Fusion.Integration.Org" Version="3.2.2" />
-    <PackageReference Include="Fusion.Integration.Roles" Version="1.1.2" />
-    <PackageReference Include="Fusion.Integration.Notification" Version="1.0.0-preview.2" />
-    <PackageReference Include="Fusion.ApiClients.Org" Version="3.1.3" />
-    <PackageReference Include="Fusion.AspNetCore" Version="3.1.7" />
-
-    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.13.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="3.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" Version="3.1.1" />
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.9.10" />
-
-    <PackageReference Include="Swashbuckle.AspNetCore.Filters" Version="5.0.0" />
-    <PackageReference Include="swashbuckle.AspNetCore.Swagger" Version="5.0.0" />
-    <PackageReference Include="swashbuckle.AspNetCore.SwaggerGen" Version="5.0.0" />
-    <PackageReference Include="swashbuckle.AspNetCore.SwaggerUi" Version="5.0.0" />
-
-    <PackageReference Include=" Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="3.1.1" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.1">
+    <PackageReference Include="FluentValidation.AspNetCore" Version="8.6.1" />
```

Comment:
There is even a version 9, but aligned with Fusion.AspNetCore version.

Reply:
Would be better to remove the local package ref if you want to align.. Would not be any problem just bumping to latest version here.

Reply:
Maybe incorrect question from my part.
Tried to bump the version to v9, but it failed when running tests.
Only guessing here, but seems like it doesn't play well with other libraries.
Using 8.6.3